index.html

```diff
@@ -204,14 +205,10 @@ <h1 class="mb-4 text-xl font-bold print:hidden">
               class="flex h-[9mm] w-[24.4mm] items-center border-[0.5mm] odd:border-blue-500 even:border-red-300"
               :class="{'even:border-transparent odd:border-transparent': !borderToggle}"
             >
-              <img
-                :src="label.qrCodeUrl"
-                alt=""
-                width="100"
-                height="100"
+              <div
+                x-effect="$el.innerHTML=''; new QRCode($el, { text: label.text, correctLevel: QRCode.CorrectLevel.L});"
```

Comment:
I think instead of `$el.innerHTML='' `, one should call `qrcode.clear()`

I think one could also call `qrcode.makeCode("...");`

This could mean we could put all this code down into a dedicated method within `qrCodeApp`